0.2.1 Broken build with PHP 7

Buildtime 

    /builddir/build/BUILD/php70-php-pecl-env-0.2.1/NTS/env.c:52:7: warning: implicit declaration of function 'Z_STRVAL_PP' [-Wimplicit-function-declaration]
      free(Z_STRVAL_PP(str));
           ^

Runtime

    PHP Warning:  PHP Startup: Unable to load dynamic library '/builddir/build/BUILDROOT/php70-php-pecl-env-0.2.1-1.fc23.remi.x86_64/opt/remi/php70/root/usr/lib64/php/modules/env.so' - /builddir/build/BUILDROOT/php70-php-pecl-env-0.2.1-1.fc23.remi.x86_64/opt/remi/php70/root/usr/lib64/php/modules/env.so: undefined symbol: Z_STRVAL_PP in Unknown on line 0

--- a/env.c
+++ b/env.c
@@ -41,18 +41,10 @@ PHP_INI_END()
 /* }}} */
 
 
-#if PHP_VERSION_ID < 70000
 void char_ptr_dtor(char **str)
 {
 	free(*str);
 }
-#else
-void char_ptr_dtor(zval **str)
-{
-	free(Z_STRVAL_PP(str));
-	free(*str);
-}
-#endif
 
 /* {{{ PHP_GINIT_FUNCTION
  */

